lang/en/docs/_ci/codefresh.md

```diff
@@ -0,0 +1,42 @@
+[Codefresh pipelines](https://codefresh.io/docs/docs/configure-ci-cd-pipeline/introduction-to-codefresh-pipelines/) use Docker images in all their steps so it is very easy to use any [Yarn version](https://hub.docker.com/_/node/) in any pipeline.
```

Comment:
I think I might be fixed by starting with an empty line here

Reply:
@Haroenv I tried adding an empty line. But there is a pre-commit hook that runs cleanups on js, md and scess files and removes it automatically :-| 
Any other ideas?

Reply:
empty line with a comment in? Sorry for the confusion here, it's definitely my fault

Reply:
@Haroenv yep that did it! I also fixed the gitlab page with the same comment.